cli/flox-rust-sdk/src/providers/services.rs

```diff
@@ -79,6 +112,44 @@ impl From<ManifestServices> for ProcessComposeConfig {
     }
 }
 
+impl Serialize for ProcessComposeConfig {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: Serializer,
+    {
+        let mut processes = self.processes.clone();
+        // Inject an extra process to prevent `process-compose` from exiting when all services are stopped.
+        processes.insert(
+            PROCESS_NEVER_EXIT_NAME.to_string(),
```

Comment:
question (nb): do we need to make sure that users don't name any of their services the same as `PROCESS_NEVER_EXIT_NAME`?

Reply:
Yes, but we have a later ticket for validating the config. Right now we're just passing everything straight through.